src/application.php

```diff
@@ -45,6 +49,13 @@
         }
 
         $configuration = readConfiguration($inputFile);
+
+        if (!$input->getOption('ignore-override') && file_exists($overrideFile)) {
+            $override = readConfiguration($overrideFile);
+
+            $configuration = array_merge_recursive($configuration, $override);
```

Comment:
The question here is should we use `array_merge_recursive` or `array_replace_recursive` ?